prac_06/car.py

```diff
@@ -0,0 +1,36 @@
+"""CP1404/CP5632 Practical - Car class example."""
+
+
+class Car:
+    """Represent a Car object."""
+
+    def __init__(self, name, fuel=0):
```

Comment:
'name' in __init__ should be: name=""

Reply:
I know the solutions use this but a name is required for each Car otherwise the __str__ function and any function referencing the name of the Car object would display weirdly. If you print the object without looking at the code you do not know what car instance it is referring to.